docs/hazmat/primitives/key-derivation-functions.rst

```diff
@@ -324,6 +324,110 @@ Different KDFs are suitable for different tasks such as:
         ``key_material`` generates the same key as the ``expected_key``, and
         raises an exception if they do not match.
 
+.. currentmodule:: cryptography.hazmat.primitives.kdf.concatkdf
+
+.. class:: ConcatKDF(algorithm, length, otherinfo, backend)
+
+    .. versionadded:: 0.9
+
+    `ConcatKDF`_ (Concatenation Key Derivation Function) is defined by the NIST
```

Comment:
This is the bit causing the test failure. Is ConcatKDF supposed to link to anything?